lua/diffview/hl.lua

```diff
@@ -121,12 +121,12 @@ function M.get_hl(name, no_trans)
   local hl
 
   if no_trans then
-    hl = api.nvim__get_hl_defs(0)[name]
+    hl = api.nvim__get_hl_defs and api.nvim__get_hl_defs(0)[name] or api.nvim_get_hl(0, {})[name]
```

Comment:
```suggestion
    hl = api.nvim__get_hl_defs and api.nvim__get_hl_defs(0)[name] or api.nvim_get_hl(0, {name=name})
```

This might be a cleaner implementation

Reply:
Thanks for the recommendation, overlooked when I committed. Updated the pr